docs/changelog.rst

```diff
@@ -4,6 +4,10 @@ Changelog
 
 *[CalVer, YY.month.patch](https://calver.org/)*
 
+24.8.1
```

Comment:
Should this be combined with the other 24.8.1 section?

Reply:
no yeah I messed up, 24.8.1 is already released >.>